src/docs/resources/platform-adaptations.md

```diff
@@ -0,0 +1,218 @@
+---
+title: Platform Specific Behaviors and Adaptations
+---
+
+## Adaptation Philosophy
+
+There are generally 3 categories of platform adaptiveness:
+
+1. Things that are behaviors of the OS environment (such as text editing and
+scrolling) and that would be 'wrong' a different behavior took place.
+2. Things that are conventionally implemented in apps using the OEM's SDKs
+(such as using parallel tabs on iOS or showing an [android.app.AlertDialog](https://developer.android.com/reference/android/app/AlertDialog.html)
+on Android).
+3. Correct usage of corresponding platform APIs to export to TalkBack/VoiceOver,
+status bar color theme etc. and to import notch metrics, locale settings etc.
+
+This article mainly covers the automatic adaptations provided by Flutter
+in case 1 on Android and iOS.
+
+For case 2, Flutter bundles the means to produce the appropriate effects of
+the platform conventions but does not adapt automatically when app design
+choices are needed. For a discussion, see [#8410](https://github.com/flutter/flutter/issues/8410#issuecomment-468034023).
+
+Flutter should transparently work for scenario 3. 
```

Comment:
You've called them categories, cases, and scenarios. Please be consistent.

Reply:
Done